init.lua

```diff
@@ -1,35 +1,19 @@
-local GameAnalytics
-if game:GetService("RunService"):IsServer() then
-	GameAnalytics = require(script:WaitForChild("gameanalytics-sdk"):WaitForChild("GameAnalytics"))
-end
-
 return {
+	GameAnalytics = require(script:WaitForChild("gameanalytics-sdk")),
+	
 	initializeClient = function()
-		require(script:WaitForChild("gameanalytics-sdk"):WaitForChild("GameAnalyticsClient"))
-	end,
-	initializeServer = function(gameKey, secretKey)
-		GameAnalytics:initialize({
-			build = "0.1",
-
-			gameKey = gameKey,
-			secretKey = secretKey,
+		if not game:GetService("RunService"):IsClient() then
+			error("initializeClient() can only be called on the client")
+		end
 
-			enableInfoLog = true,
-			enableVerboseLog = false,
-
-			--debug is by default enabled in studio only
-			enableDebugLog = nil,
+		return require(script:WaitForChild("gameanalytics-sdk")).initClient()
+	end,
 
-			automaticSendBusinessEvents = true,
-			reportErrors = true,
+	initializeServer = function(gameKey: string, secretKey: string)
+		if not game:GetService("RunService"):IsServer() then
+			error("initializeServer() can only be called on the server")
+		end
 
-			availableCustomDimensions01 = {},
-			availableCustomDimensions02 = {},
-			availableCustomDimensions03 = {},
-			availableResourceCurrencies = {},
-			availableResourceItemTypes = {},
-			availableGamepasses = {},
-		})
-	end,
-	GameAnalytics = GameAnalytics,
+		return require(script:WaitForChild("gameanalytics-sdk")).initServer(gameKey, secretKey)
```

Comment:
Careful, because I ran `initializeServer` and GA errored because the secretKey argument was missing. Turns out this is a method:
```suggestion
		return require(script:WaitForChild("gameanalytics-sdk")):initServer(gameKey, secretKey)
```